server/src/main/java/org/opensearch/telemetry/tracing/AttributeNames.java

```diff
@@ -94,4 +94,19 @@ private AttributeNames() {
      * Refresh Policy
      */
     public static final String REFRESH_POLICY = "refresh_policy";
+
+    /**
+     * Search Request Source
+     */
+    public static final String SOURCE = "source";
+
+    /**
+     * Search Response Shard Stats
+     */
+    public static final String SHARDS = "shards";
```

Comment:
Looking the examples from the description:

```
{
         "key":"total_hits",
         "value":{
            "stringValue":"4 hits"
         }
      },
      {
         "key":"shards",
         "value":{
            "stringValue":"{total:2, successful:2, skipped:0, failed:0}"
         }
      }
```

This is not how the attributes should be modelled, please keep in mind the attributes are searchable:
- `total_hits` should be a number: fe, the user could easily find a trace `> 1000` hits
- `shards` should be split into separate attributes for the same reasons (`total_shards`, `failed_shards`, ...) so the attributes could be queriable